examples/tls-inspector/verify.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+export NAME=tls-inspector
+
+# shellcheck source=examples/verify-common.sh
+. "$(dirname "${BASH_SOURCE[0]}")/../verify-common.sh"
+
+run_log "Test tls inspector"
```

Comment:
can you change this to something like 

```
 run_log "Curl tls inspector: HTTPS -> HTTP/1.1"
```
and then also do similar before each of the following commands 

it means that if the test fails in CI its much easier to find exactly which command failed

it also makes it more useful running and seeing what is being tested

(apologies for last minute nits!)

Reply:
Hah, don't worry about that. I'm glad to make it looks better, thanks for your comments! :smile: